playbooks/aws/openshift-cluster/vars.yml

```diff
@@ -31,3 +31,100 @@ deployment_vars:
   enterprise: "{{ deployment_rhel7_ent_base }}"
   openshift-enterprise: "{{ deployment_rhel7_ent_base }}"
   atomic-enterprise: "{{ deployment_rhel7_ent_base }}"
+
+provision:
+  cert_arn: "arn:aws:iam::000000000000:server-certificate/<clusterid>_<domain>"
+  iam_cert_ca:
+    name: test_openshift
+    cert_path: '/path/to/wildcard.<clusterid>.example.com.crt'
+    key_path: '/path/to/wildcard.<clusterid>.example.com.key'
+    chain_path: '/path/to/cert.ca.crt'
+  yum_repositories:
```

Comment:
we probably shouldn't nest all these dictionaries in provision.  That would require someone wanting to override provision itself to hand-set every single dictionary here.  Some of these could be their own top-level variable, such as
"provision_yum_repos"